analyzer/fx/src/main/resources/jp/co/ntt/oss/heapstats/plugin/builtin/snapshot/tabs/histogram.fxml

```diff
@@ -67,10 +67,10 @@
                 </VBox>
                 <VBox maxHeight="1.7976931348623157E308" maxWidth="1.7976931348623157E308" minHeight="0.0" minWidth="0.0" spacing="5.0">
                     <children>
-                        <HBox spacing="5.0">
+                        <HBox maxWidth="1.7976931348623157E308" spacing="5.0">
```

Comment:
This value is from `MAX_VALUE` in Scene Builder.
You can check it on Scene Builder.